src/coverage.ts

```diff
@@ -14,8 +14,8 @@ import {
 import * as os from "os";
 import * as nativePath from "path";
 import * as ChildProcess from "child_process";
-import {SourceMap} from "./parser.c";
-import {GcovData, loadGcovData} from "./gcov";
+import { SourceMap } from "./parser.c";
```

Comment:
Although I am all for this kind of readability changes, we sometimes want to be able to grab fixes from the original/forked repository.
To do that more easily, unfortunately we need to avoid any unnecessary change, even white-space ones.